saleor/graphql/schema.graphql

```diff
@@ -3446,6 +3442,7 @@ enum ProductErrorCode {
   REQUIRED
   UNIQUE
   VARIANT_NO_DIGITAL_CONTENT
+  PRODUCT_WITHOUT_VARIANT
```

Comment:
This code could be more descriptive, e.g. `CANNOT_MANAGE_PRODUCT_WITHOUT_VARIANT`.

Reply:
We need to create a ticket for the frontend team to handle this error in the dashboard.